core/templates/dev/head/AppSpec.ts

```diff
@@ -16,8 +16,20 @@
  * @fileoverview Unit tests for generic services.
  */
 
+// TODO(YashJipkate) Remove the unnnecessary imports once the corresponding
```

Comment:
In your TODOs here and elsewhere, be more specific: it's not clear which imports are "unnecessary" and which are not. It is also not clear which "non-spec file" is being referred to.

Please say something like "Remove the following two imports" and also specify the non-spec files explicitly. Also say "upgraded to Angular 8" rather than just "upgraded".

In general, TODOs should be really specific.

Reply:
I was thinking more like having a block notation like we have for webpack requires, and separate the 'needed' imports from the unneeded ones by a newline. At this point of time there is no 'needed' import (in the files not upgraded), since the imports arise only because of the Angular 8 dependency.
Also, the 'unnecessary' imports are only required in spec files, so the corresponding non-spec file refers to the file that's test resides in that spec file, should I name something more appropriate or should I explicitly lay out the name of the file?

Reply:
If you go with a block notation, there should be a comment at the end of the block denoting the "end" of the "unnecessary" imports.

Also, please explicitly lay out the name of the file. It's unclear whether "corresponding" refers to the file we're in, or the file that's being imported, or something else.

Reply:
Done!